moveit_experimental/jog_arm/test/arm_setup/urdf/test_six_dof_arm.xacro

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://www.ros.org/wiki/xacro" name="six_dof_arm" >
+    <xacro:include filename="$(find jog_arm)/test/arm_setup/urdf/test_six_dof_arm_macro.xacro"/>
```

Comment:
What robot xacro and stl are you committing? Can you use one of the 3 robots already available in https://github.com/ros-planning/moveit_resources ? If not, please add this robot to moveit_resources or consider using @BryceStevenWilley 's new test robot framework:
https://ros-planning.github.io/moveit_tutorials/doc/tests/tests_tutorial.html

Reply:
The test framework is meant for unit test size stuff, it's probably not the best solution for this size of integration test.